yw_website/yw_website/apps/website/static/design/search.css

```diff
@@ -0,0 +1,15 @@
+.search {
+    margin: 0;
+    font-family: Verdana;
+    text-align: justify;
+    border-spacing: 0 1em;
+}
```

Comment:
This file is good. We should stop applying styling to tr and ul's it messes with other pages.